atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/TracingPrefsConfig.java

```diff
@@ -44,9 +44,11 @@
     public void run() {
         try {
             final File TRACING_PREF_FILE = new File(System.getProperty("user.dir") + java.io.File.separatorChar + TRACING_PREF_FILENAME);
+
             if (TRACING_PREF_FILE.exists()) {
+                FileInputStream fileStream = new FileInputStream(TRACING_PREF_FILE);
                 try {
```

Comment:
I don't think atlasdb-client has to have source/target 1.6, but I think commons does for now (though we don't enforce it here, need to fix up gradle and tests).